frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -17,13 +20,50 @@ import WalletLink from "../utils/WalletLink";
 import { Translate } from "react-localize-redux";
 
 export interface Props {
-  account: A.Account;
+  account: Account;
   currentNearNetwork: NearNetwork;
 }
 
+interface State {
+  transactionCount?: AccountStats;
+}
+
 class AccountDetails extends React.Component<Props> {
+  state: State = {
+    transactionCount: {},
```

Comment:
I would remove it completely or define as `undefined` to match the `State` interface.

```suggestion
```